modules/gridss/1.0/gridss.smk

```diff
@@ -388,6 +483,7 @@ rule _gridss_filter_gripss:
     conda: 
         CFG["conda_envs"]["bcftools"]
     shell: 
+        # 'touch {output} && sleep 15'
```

Comment:
Why is this here?

Reply:
This was group job debugging, will delete throughout.